pkg/retry/azure_retry.go

```diff
@@ -201,3 +215,31 @@ func delayForBackOff(backoff *Backoff, cancel <-chan struct{}) bool {
 		return false
 	}
 }
+
+func isRetriableErrors(rerr *Error, nonRetriableErrors []string) bool {
```

Comment:
Ditto. And is it confused that L226 returns true if the given error is one of the **Non-retriable** errors, but the naming here indicates that it should be **retriable** instead?

Reply:
updated